adafruit_pyportal.py

```diff
@@ -51,15 +51,10 @@
 from digitalio import DigitalInOut
 import pulseio
 import neopixel
-try:
-    import adafruit_touchscreen
-except ImportError:
-    pass
-try:
-    from adafruit_cursorcontrol.cursorcontrol import Cursor
-    from adafruit_cursorcontrol.cursorcontrol_cursormanager import CursorManager
-except ImportError:
-    pass
+
+import adafruit_touchscreen
+from adafruit_cursorcontrol.cursorcontrol import Cursor
```

Comment:
dont forget to go thru the pyportal guide(s) to tell them to install this library as well

Reply:
Guides mirror https://learn.adafruit.com/adafruit-pyportal/pyportal-circuitpython-setup, added cursorcontrol and desc. to that page.